tests/common/test_cgroupapi.py

```diff
@@ -27,6 +27,8 @@
 from azurelinuxagent.common.future import ustr
 from azurelinuxagent.common.utils import shellutil, fileutil
 from nose.plugins.attrib import attr
+
+from azurelinuxagent.common.utils.extensionprocessutil import TELEMETRY_MESSAGE_MAX_LEN
```

Comment:
At some point I think I clicked on "format file" so there's a lot of whitespace updates, I'll point out relevant updates to the tests.

Reply:
so we need to review only the lines where you added comments?

Reply:
For this file, that's the case, yes.

Reply:
your difftool should allow you selectively undo the changes you don't want (meld does). In the future let's try to avoid global formatting changes, since it makes the review and reading the history of the code harder,

Reply:
(you can even undo the changes after commiting, if you feel like doing it here)